content/doc/pipeline/index.adoc

```diff
@@ -599,14 +599,17 @@ a build tool):
 [source, groovy]
 ----
  node {
-   git url: 'https://github.com/joe_user/simple-maven-project-with-tests.git'
+   git 'https://github.com/joe_user/simple-maven-project-with-tests.git'
    def mvnHome = tool 'M3'
    sh "${mvnHome}/bin/mvn -B -Dmaven.test.failure.ignore verify"
-   step([$class: 'ArtifactArchiver', artifacts: '**/target/*.jar', fingerprint: true])
-   step([$class: 'JUnitResultArchiver', testResults: '**/target/surefire-reports/TEST-*.xml'])
+   archiveArtifacts artifacts: '**/target/*.jar', fingerprint: true
+   junit '**/target/surefire-reports/TEST-*.xml'
```

Comment:
FYI, this is the result of `JUnitResultArchiver`

* being a `SimpleBuildStep`
* having a `@Symbol` on its `Descriptor`
* having a one-argument `@DataBoundConstructor`